ui/raidboss/data/04-sb/raid/o9s.js

```diff
@@ -185,20 +185,20 @@
     {
       id: 'O9S Fire Phase Tracking',
       regex: / 14:3186:Chaos starts using Blaze/,
-      regexDe: / 14:3186:Chaos starts using /,
-      regexFr: / 14:3186:Chaos starts using /,
-      regexJa: / 14:3186:カオス starts using /,
+      regexDe: / 14:3186:Chaos starts using Flamme/,
+      regexFr: / 14:3186:Chaos starts using Flammes/,
+      regexJa: / 14:3186:カオス starts using ほのお/,
       run: function(data) {
         if (data.phaseType != 'enrage')
           data.phaseType = 'fire';
       },
     },
     {
       id: 'O9S Entropy Spread',
-      regex: /1A:\y{ObjectId}:(\y{Name}) gains the effect of (?:Unknown_640|Entropy) from  for (\y{Float}) Seconds/,
-      regexDe: /1A:\y{ObjectId}:(\y{Name}) gains the effect of (?:Unknown_640|Chaosflammen) from  for (\y{Float}) Seconds/,
-      regexFr: /1A:\y{ObjectId}:(\y{Name}) gains the effect of (?:Unknown_640|Flammes du chaos) from  for (\y{Float}) Seconds/,
-      regexJa: /1A:\y{ObjectId}:(\y{Name}) gains the effect of (?:Unknown_640|混沌の炎) from  for (\y{Float}) Seconds/,
+      regex: / 1A:\y{ObjectId}:(\y{Name}) gains the effect of Entropy from .* for (\y{Float}) Seconds/,
+      regexDe: / 1A:\y{ObjectId}:(\y{Name}) gains the effect of Chaosflammen from .* for (\y{Float}) Seconds/,
+      regexFr: / 1A:\y{ObjectId}:(\y{Name}) gains the effect of Flammes du chaos from .* for (\y{Float}) Seconds/,
+      regexJa: / 1A:\y{ObjectId}:(\y{Name}) gains the effect of 混沌の炎 from .* for (\y{Float}) Seconds/,
```

Comment:
Oh! For some reason I had misremembered that yName only handled things with one space, but I was wrong.  This sounds like a great idea.

(Although honestly part of me wonders if yName should just resolve to `.*?` <_<)

Reply:
I asked this question as well on the day were i worked on these triggers xD

So want it to be replaced with \y{Name} now?

Reply:
I think the yName's regex is a little daunting from a grokking perspective, so `.*?` is probably _fine_. Notably, the current regex does not match on anything like `[](*&!^@,`, with the only one of those we actually care about probably being the comma, and _maybe_ the parentheses? So, its current state is actually deficient from what all would need to be matched, but it can easily be updated to include things as they come up, notably Dohn Mheg and bosses with commas in their name...

Reply:
(Personally, I think a goal would be to include as few `.*` within the exposed regexes as possible, even if we're masquerading them with things like `\y{Name}`.)

Reply:
Don't know if this makes sense but for readability purpose i would suggest to use 3 variables for Player, Enemy and Ability to keep regex behind as detailed as possible to avoide potential mismatches and also make the regex as easy understandable as possible (but probably forgetting something again and getting scold by @quisquous for this again xD)

Reply:
Ok will leave them as they currently are. Will mark this as resolved for now.